app/epa/settings.py

```diff
@@ -1,255 +1,259 @@
+    }
+}
+
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
+
+# Password validation
+# https://docs.djangoproject.com/en/3.0/ref/settings/#auth-password-validators
+
+AUTH_PASSWORD_VALIDATORS = [
+    {"NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator"},
+    {"NAME": "django.contrib.auth.password_validation.MinimumLengthValidator"},
+    {"NAME": "django.contrib.auth.password_validation.CommonPasswordValidator"},
+    {"NAME": "django.contrib.auth.password_validation.NumericPasswordValidator"},
+]
+
+# Internationalization
+# https://docs.djangoproject.com/en/3.0/topics/i18n/
+
+LANGUAGE_CODE = "en"
+
+LOCALE_PATHS = (os.path.join(BASE_DIR, "locale"),)
+
+LANGUAGES = [("de", "German"), ("en", "English")]
+
+TIME_ZONE = "Europe/Copenhagen"
+
+USE_I18N = True
+
+USE_L10N = True
+
+USE_TZ = False
+
+# Other configs
+
+AUTH_USER_MODEL = "users.CustomUser"
+
+LOGIN_URL = "login"
+LOGIN_REDIRECT_URL = "home_cpn"
+LOGOUT_REDIRECT_URL = "home_cpn"
+
+CRISPY_TEMPLATE_PACK = "bootstrap4"
+
+# Please note, we don't use Django's internal email system,
+# we implement our own, using exchangelib
+USE_EXCHANGE_EMAIL_BACKEND = ast.literal_eval(os.getenv("USE_EXCHANGE_EMAIL_BACKEND", "True"))
+# The Exchange account which sends emails
+EXCHANGE_ACCOUNT = os.getenv("EXCHANGE_ACCOUNT", "dummy@dummy.com")
+EXCHANGE_PW = os.getenv("EXCHANGE_PW", "dummypw")
+EXCHANGE_EMAIL = os.getenv("EXCHANGE_EMAIL", "dummy@dummy.com")
+EXCHANGE_SERVER = os.getenv("EXCHANGE_SERVER", "dummy.com")
+# Email addresses to which feedback emails will be sent
+RECIPIENTS = os.getenv("RECIPIENTS", "dummy@dummy.com,dummy2@dummy.com").split(",")
+EMAIL_SUBJECT_PREFIX = os.getenv("EMAIL_SUBJECT_PREFIX", "[open_plan] ")
+
+MESSAGE_TAGS = {
+    messages.DEBUG: "alert-info",
+    messages.INFO: "alert-info",
+    messages.SUCCESS: "alert-success",
+    messages.WARNING: "alert-warning",
+    messages.ERROR: "alert-danger",
+}
+
+USE_PROXY = ast.literal_eval(os.getenv("USE_PROXY", "True"))
+PROXY_ADDRESS_LINK = os.getenv("PROXY_ADDRESS", "http://proxy:port")
+PROXY_CONFIG = ({"http://": PROXY_ADDRESS_LINK, "https://": PROXY_ADDRESS_LINK}) if USE_PROXY else ({})
+
+MVS_API_HOST = os.getenv("MVS_API_HOST", "https://mvs-eland.rl-institut.de")
+MVS_POST_URL = f"{MVS_API_HOST}/sendjson/"
+MVS_GET_URL = f"{MVS_API_HOST}/check/"
+MVS_LP_FILE_URL = f"{MVS_API_HOST}/get_lp_file/"
+MVS_SA_POST_URL = f"{MVS_API_HOST}/sendjson/openplan/sensitivity-analysis"
+MVS_SA_GET_URL = f"{MVS_API_HOST}/check-sensitivity-analysis/"
+
+# Allow iframes to show in page
+X_FRAME_OPTIONS = "SAMEORIGIN"
+
+# API key to fetch exchange rates
+EXCHANGE_RATES_API_KEY = os.getenv("EXCHANGE_RATES_API_KEY")
+EXCHANGE_RATES_URL = f"https://v6.exchangerate-api.com/v6/{EXCHANGE_RATES_API_KEY}/latest/USD"
+
+# API to connect with KoboToolbox
+KOBO_API_URL = "https://kf.kobotoolbox.org/api/v2/"
+KOBO_API_TOKEN = os.getenv("KOBO_API_TOKEN")
```

Comment:
Somehow it is showing the entire files as changed even though I am only committing bits of code. Not sure why that is. Sorry about that. This is what is actually new.